index.js

```diff
@@ -19,6 +19,19 @@ server.get('/api/cohorts', async (req, res) => {
   }
 });
 
```

Comment:
recommend you to put an error return / catch for when a required field is not put it, maybe before your try block, and return an error code if there is a missing name field